### ISSUE TITLE
Organizing Chinese community

### ISSUE THREAD

**Issue by scvoet** [flutter/flutter#38805]
> The following is not related to Flutter development.

Today, I was amazed to find that many people in the world are doing **repetitive** work. They built many similarly website to share Flutter, but it was destined to be **extremely inefficient**.
Below I list some known Flutter Chinese communities.

1. https://www.cflutter.com/
2. https://flutterchina.club/
3. https://fluttercc.com/
4. https://flutter.axuer.com/
5. https://www.flutterchina.net.cn/
6. http://flutter.blackdir.com/
7. http://studyflutter.com/
8. http://flutter.link/
9. https://code4flutter.com/
10. http://flutter.liulongbin.top/
11. https://www.fluttercn.net/
12. http://www.flutterj.com/
13. https://www.flutter123.net/
14. https://flutter-io.cn/
There are more sites we don't know.

It's easy to find that there are so many similar sites that don't make sense.
These sites can be divided into three categories: document translation, forums, and tutorials.
These little-meaning websites are scattered by Flutter Chinese developers.
I think if there is a cohesive community, then the Flutter Chinese ecology will be more prosperous.

So I hope that **the official of Flutter officially organizes a Chinese community** or **the webmasters of the Chinese community merged the original community together.**.

I hope that Chinese developers can support my initiative.

**Comment by jmagman** [comment]
@zoeyfan

**Comment by BigCoke233** [comment]
I agree. But I don't think the official is glad to organize a community only for Chinese.
They can provide more language versions besides Chinese for users to choose in the original community even if the content of the post can't be translated. Yet I think most of Chinese developers can understand it and will be satisfied with the Chinese operation interface.

---

我同意你的观点。但是我不认为官方会乐意专门为了华人组织一个社区。
他们可以在社区里提供更多包括中文的语言版本供用户选择，即使帖子内容不能被翻译。
但我觉得大部分华人开发者都能理解，并且会对中文的操作界面感到满意。

**Comment by scvoet** [comment]
> I agree. But I don't think the official is glad to organize a community only for Chinese.
> They can provide more language versions besides Chinese for users to choose in the original community even if the content of the post can't be translated. Yet I think most of Chinese developers can understand it and will be satisfied with the Chinese operation interface.
> 
> 我同意你的观点。但是我不认为官方会乐意专门为了华人组织一个社区。
> 他们可以在社区里提供更多包括中文的语言版本供用户选择，即使帖子内容不能被翻译。
> 但我觉得大部分华人开发者都能理解，并且会对中文的操作界面感到满意。

Regardless of the last, the fragmented will only cause confusion.
---
不管最后怎么样，各自为政只会造成混乱。
